dateparser/custom_language_detection/langdetect.py

```diff
@@ -0,0 +1,32 @@
+from langdetect.detector_factory import DetectorFactory, PROFILES_DIRECTORY
+import langdetect
+
+
```

Comment:
I would love to see here a comment pointing to the lines of code of the langdetect project, and explaining why we do this (`init_factory`, etc)

Reply:
Thanks, I have added some comments there